src/main/java/frc/robot/RobotContainer.java

```diff
@@ -7,92 +7,151 @@
         JoystickButton btnAlign = new JoystickButton(leftJoystick, 1);
         JoystickButton btnIntakeIn = new JoystickButton(altJoystick, 3);
         JoystickButton btnIntakeOut = new JoystickButton(altJoystick, 4);
-        JoystickButton btnCPExtend = new JoystickButton(leftJoystick, 4); // toggle: use "toggleWhenPressed" method to set command
+        JoystickButton btnCPExtend = new JoystickButton(leftJoystick,
+            4); // toggle: use "toggleWhenPressed" method to set command
         JoystickButton btnWinch = new JoystickButton(altJoystick, 8);
         JoystickButton btnCPMotor = new JoystickButton(leftJoystick, 3);
         JoystickButton btnLauncherMotor = new JoystickButton(altJoystick, 12);
-        JoystickButton btnLauncherIdle = new JoystickButton(altJoystick, 10); // toggle: use "toggleWhenPressed" method to set command
+        JoystickButton btnLauncherIdle = new JoystickButton(altJoystick,
+            10); // toggle: use "toggleWhenPressed" method to set command
         JoystickButton btnLauncherMotorClose = new JoystickButton(altJoystick, 11);
         JoystickButton btnLauncherMotorDynamic = new JoystickButton(altJoystick, 9);
         JoystickButton btnSlowMovement = new JoystickButton(rightJoystick, 1);
-        JoystickButton btnIntakeSolenoid = new JoystickButton(altJoystick, 2); // toggle: use "toggleWhenPressed" method to set command
+        JoystickButton btnIntakeSolenoid = new JoystickButton(altJoystick,
+            2); // toggle: use "toggleWhenPressed" method to set command
         JoystickButton btnScissorExtend = new JoystickButton(altJoystick, 7);
         JoystickButton btnColorSensor = new JoystickButton(leftJoystick, 5);
         JoystickButton btnCPStop = new JoystickButton(leftJoystick, 2);
         JoystickButton btnInvertYAxis = new JoystickButton(leftJoystick, 6);
         JoystickButton btnRotationSensitivity = new JoystickButton(rightJoystick, 1);
         JoystickButton btnIntakeBottomOut = new JoystickButton(altJoystick, 6);
 
-        driveSubsystem.setDefaultCommand(new RunCommand(() -> driveSubsystem.drive(-leftJoystick.getY() * 0.7, rightJoystick.getX() * 0.7), driveSubsystem));
+        driveSubsystem.setDefaultCommand(new RunCommand(
+            () -> driveSubsystem.drive(-leftJoystick.getY() * 0.7, rightJoystick.getX() * 0.7),
+            driveSubsystem));
         shooterSubsystem.setDefaultCommand(new RunCommand(() -> {
             shooterSubsystem.shoot(Math.abs(altJoystick.getY()));
             System.out.println("Spinning");
         }, shooterSubsystem));
 
-        btnLauncherSolenoid
-            .whenPressed(new InstantCommand(() -> shooterSubsystem.activatePiston(), shooterSubsystem))
-            .whenInactive(new InstantCommand(() -> shooterSubsystem.lowerPiston(), shooterSubsystem));
+        btnLauncherSolenoid.whenPressed(
+            new InstantCommand(shooterSubsystem::activatePiston, shooterSubsystem))
```

Comment:
Are you guys familiar with this syntax? The ctor for `Instantcommand` takes a method as its first arguement. You can create an on-the-fly method (a lambda) or you can pass it in a method. In python you could just do `shooterSubsystem.activatePiston` but in java you use the `::` operator to get a method reference.

Reply:
I think sticking with the arrow would probably be better because it's similar in nature to arrow functions in javascript and a little more intuitive.